http_server.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"net/http"
+
+	"fmt"
+)
+
+
+
+
+func main() {
+	fmt.Println("staring server")
+
+
+	http.HandleFunc("/", page )
+	http.ListenAndServe(":5002",nil)
+
+}
+func page(writer http.ResponseWriter, request *http.Request) {
+	fmt.Fprint(writer,`<!DOCTYPE html>
```

Comment:
You can shift this html code to separate file and use here as variable so it may look much nicer

Reply:
i was trying to do it through template but i was getting an error of file not found. may be this was because of path error so i did it in this way